.github/workflows/inspektor-gadget.yml

```diff
@@ -677,25 +677,25 @@ jobs:
         filters: |
           pattern: ${{ matrix.image.filter-pattern }}
     - name: Set up Docker Buildx
-      if: steps.filter.outputs.pattern == 'true'
+      if: steps.filter.outputs.pattern == 'true' || startsWith(github.ref, 'refs/tags/v')
```

Comment:
Would it make sense to define a new `should-skip` step with this logic and then other steps will only check a single output from the `should-skip` step?